frontend/src/models/Federation.model.ts

```diff
@@ -90,9 +91,12 @@ export class Federation {
   };
 
   updateUrl = async (origin: Origin, settings: Settings, hostUrl: string): Promise<void> => {
+    const federationUrls = {};
     for (const coor of Object.values(this.coordinators)) {
       coor.updateUrl(origin, settings, hostUrl);
+      federationUrls[coor.shortAlias] = coor.url;
     }
+    systemClient.setCookie('federation', JSON.stringify(federationUrls));
```

Comment:
Informs the system the current coordinators and their URLs, useful when changing to tesnet and to avoid duplication on the system